sys/include/net/sock/dns.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_sock_dns    DNS sock API
+ * @ingroup     net_sock
+ *
+ * @brief       Sock DNS client
+ *
+ * @{
+ *
+ * @file
+ * @brief   DNS sock definitions
+ *
+ * @author  Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef SOCK_DNS_H
+#define SOCK_DNS_H
+
+#include <errno.h>
+#include <stdint.h>
+#include <unistd.h>
+
+#include "net/sock/udp.h"
+#include "arpa/inet.h"
```

Comment:
1. This is a POSIX header, so it should be included with `#include <arpa/inet.h>`.
2. Why is it included in the first place?